assignment08/js/clock.js

```diff
@@ -0,0 +1,87 @@
+window.addEventListener('load', () => {
+ class Clock   {   
+
+startClock ()  {
+    //YOUR CODE SHOULD START BELOW THIS LINE
+const clock = document.getElementById('clock');
+const context = clock.getContext('2d');
+let radius = clock.height / 2;
+context.translate(radius, radius);
+radius = radius * 0.90;
+drawClock();
+
+function drawClock(){
+    drawFace(context, radius);
+    drawNumbers(context, radius);
+    drawTime(context, radius);
+}
+function drawFace(context, radius) {
+    let grad;
+    context.beginPath();
+    context.arc(0, 0, radius, 0, 2 * Math.PI);
+    context.fillStyle = 'white';
+    context.fill();
+
+    grad = context.createRadialGradient(0, 0, radius * 0.95, 0, 0, radius * 1.05);
+    grad.addColorStop(0, '#333');
+    grad.addColorStop(0.5, 'white');
+    grad.addColorStop(1, '#333');
+    context.strokeStyle = grad;
+    context.lineWidth = radius * 0.1;
+    context.stroke();
+
+    context.beginPath();
+    context.arc(0, 0, radius, 0, 2 * Math.PI);
```

Comment:
this line isn't quite right. You are not specifying a portion of the radius you want to draw - you are covering the whole inside of your clock with white (covering those numbers you draw). I tested this by changing your next line to a different color. I think the tutorial shows radius * 0.1 and then the fillstyle below should be #333